src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -46,8 +46,12 @@ public AddCommand(String name,
         for (String tagName : tags) {
             tagSet.add(new Tag(tagName));
         }
+
+        String backName = name.substring(1);
+        String cappedFirstLetter = Character.toString(name.charAt(0)).toUpperCase();
+        String cappedName = cappedFirstLetter + backName;
```

Comment:
It would be better to combine the code above instead of initializing 3 more Strings.
name = name.substring(0,1).toUpperCase() + name.substring(1);

Reply:
Unsafe operation as `name' is not checked to be nonempty yet. This needs it to have at least one character.

Just add a simple check 
`if (!name.isEmpty()) {
    ...
}`
or better, defer this operation to after the name has been passed into the `Person' constructor and validated.